android/src/main/java/com/swmansion/reanimated/nodes/ConcatNode.java

```diff
@@ -15,9 +17,16 @@ public ConcatNode(int nodeID, ReadableMap config, NodesManager nodesManager) {
   @Override
   protected String evaluate() {
     StringBuilder builder = new StringBuilder();
+    NumberFormat formatter = NumberFormat.getInstance();
```

Comment:
maybe avoid instantiating formatter on each eval. It seem like this could be a static object

Reply:
Moved to a static variable & block